README.md

```diff
@@ -1,42 +1,50 @@
-- [ ] PBR Texture assets
-- [ ] Brush assets for texture painting
-- [ ] Different channel presets for NPR, PBR, ...
-
-## Changelog
-#### v2.0.0
-- Initial release
+# Layer Painter
+
+Layer Painter is a addon for blender, made to bring a workflow similar to a tool like substance painter or armor paint directly inside blender.
+The goal here is to provide a workflow which makes the tools mentioned above unnecessary for a lot of projects, keeping the work fully inside of blender.
+
+**[Discord Server](https://discord.com/invite/s9dawzV5JU)**
+
+The discord server is used as the main place to share information and work regarding the addon. You can join it to ask questions and get more information about contributing.
+
+## Overview
+- [Addon](#addon)
+- [Additional Assets](#additional-assets)
+- [Tutorials](#tutorials)
+- [Contributors](#contributors)
+- [Changelog](#changelog)
+
+## Addon
+WIP (This section will contain a written description on how to use the addon)
+
+## Additional Assets
+- [Texture Based Grunge Maps by BongoCat - Already Included](https://drive.google.com/file/d/11sJSfT7jJohLHouMSQB5oc4GzUHtOifO/view)
+
+## Tutorials
+[Layer Painter 2, Complete Tutorial and free scene by blanchsb](https://www.youtube.com/watch?v=YDt9LFbjJU4)
+
+[Layer Painter 2, Materials and Asset Browser Tutorial by blanchsb](https://www.youtube.com/watch?v=glMGVkGhtAg)
+
+## Contributors
+This addon is maintained by contributors. This most importantly includes contributions for code but also for assets like masks and filters and tutorials. If you're interested in contributing any of these, just open a pull request for adding them yourself or join the discord server and ask for more information.
+
+### Code Contributions
+You can fix bugs, add your own feature ideas or other people ideas. If it's something bigger ask on the discord server before you start working to know if someone else already started on it or if you should make changes to your initial idea. When you're done, submit a pull request and your code can be added.
+
+#### Open features
+- [ ] Baking different masks to images (Bevel, Pointiness, Curvature, ...)
+- [ ] Baking masks / mask stacks to a single image mask
+- [ ] Other layer types (for example a paint only layer or a material layer)
+- [ ] PBR Texture assets
+- [ ] Brush assets for texture painting
+- [ ] Different channel presets for NPR, PBR, ...
+
+## Changelog
+#### v2.0.0
+- Initial release
+
+#### v2.0.2
+ - Added 2, Cycles Only, Edge/Cavity Mask: 'Edge and Corner Mask - Bevel C', 'Edge Mask Pointiness C' 
+ - Added 1, Eevee and Cycles Edge/Cavity Mask:'Cavity, Crevice, Innercorner Mask - from AO'
+ - Added 10 new Procedural Grunge Masks for Eevee and Cycles "Procedural Noise Masks": 'Gneiss, Malachite, Agate', 'Rust', 'Concrete/Marble', 'Dusty, 'Dirt, 'Smuged Dirt', 'Dirt Spots', 'Distorted Spots', 'Spline Noise', 'Noisy Lines'
+- Note: Masks appended with C are Cycles only. 

```

Comment:
Would be better if they were prefixed with "(Cycles)" or "Cycles - Mask Name" so that you can see this more clearly.
Alternatively a C or Cycles on the thumbnail would also work. Just a C is a bit unclear

Reply:
Due to the lack of word wrap, and the delay on hover reveal it would seem to work better on the thumbnails itself than in the name. I mean, before the change in zoom of the pop-up, all you would have seen is basically (Cycles) Mas